docs/reference/indices/rollover-index.asciidoc

```diff
@@ -84,6 +84,58 @@ POST /my_alias/_rollover/my_new_index_name
 // TEST[s/^/PUT my_old_index_name\nPUT my_old_index_name\/_alias\/my_alias\n/]
 
 [float]
+=== Using date math with the rolllover API
+
+It can be useful to use <<date-math-index-names,date math>> to name the
+rollover index according to the date that the index rolled over, e.g.
+`logstash-2016.02.03`.  The rollover API supports date math by storing the
+original `provided_name` in the index settings.  For instance:
+
+[source,js]
+--------------------------------------------------
+PUT /<logs-{now/d}> <1>
+{
+  "aliases": {
+    "logs_write": {}
+  }
+}
+
+# Wait for a day to pass
+
+POST /logs_write/_rollover <2>
+{
+  "conditions": {
+    "max_age":   "1d"
+  }
+}
+--------------------------------------------------
+// CONSOLE
+// TEST[skip:Requires 24h to pass]
+<1> Creates an index named (e.g.) `logs-2016.10.31`
+<2> Rolls over to a new index with today's date, e.g. `logs-2016.11.01`
```

Comment:
sorry this is not how it works you have to create an index `<logs-{now/d}-1>` first and then when you rollover we increment the counter if it's a new day or not. So I think this entire writeup is wrong ?